### PR TITLE
Add template core events to header of search_results.html

### PR DESCRIPTION
**Pull request by Wolfsblvt**
Adding core template events inside the header in search_results.html
for prosilver and subsilver2 to three usable places.

PHPBB3-13533

### CODE REVIEW

**Comment by Wolfsblvt** [comment]
Changed commit message to match standard.

File: phpBB/styles/prosilver/template/search_results.html [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
 <!-- IF SEARCHED_QUERY --> <p>{L_SEARCHED_QUERY}{L_COLON} <strong>{SEARCHED_QUERY}</strong></p><!-- ENDIF -->
 <!-- IF IGNORED_WORDS --> <p>{L_IGNORED_TERMS}{L_COLON} <strong>{IGNORED_WORDS}</strong></p><!-- ENDIF -->
 <!-- IF PHRASE_SEARCH_DISABLED --> <p><strong>{L_PHRASE_SEARCH_DISABLED}</strong></p><!-- ENDIF -->
 
 <!-- IF SEARCH_TOPIC -->
-	<p><a class="arrow-{S_CONTENT_FLOW_BEGIN}" href="{U_SEARCH_TOPIC}">{L_RETURN_TO_TOPIC}</a></p>
+	<p class="return-link"><a class="arrow-{S_CONTENT_FLOW_BEGIN}" href="{U_SEARCH_TOPIC}">{L_RETURN_TO_TOPIC}</a></p>
 <!-- ELSE -->
-	<p><a class="arrow-{S_CONTENT_FLOW_BEGIN}" href="{U_SEARCH}" title="{L_SEARCH_ADV}">{L_GO_TO_SEARCH_ADV}</a></p>
+	<p class="advanced-search-link"><a class="arrow-{S_CONTENT_FLOW_BEGIN}" href="{U_SEARCH}" title="{L_SEARCH_ADV}">{L_GO_TO_SEARCH_ADV}</a></p>
```
Comment (nickvergessen) [inline phpBB/styles/prosilver/template/search_results.html]: please remove the trailing whitespace

**Comment by bantu** [comment]
The commit message of the first commit does not follow our template: https://wiki.phpbb.com/Git#Commit_Messages

**Comment by nickvergessen** [comment]
Merged via https://github.com/phpbb/phpbb/compare/c24a84004823...461ce3f89249

**Pull request closed without merging**